package.json

```diff
@@ -2,18 +2,15 @@
   "name": "spaniel",
   "version": "1.0.0",
   "description": "LinkedIn's viewport tracking library",
-  "license" : "Apache 2.0",
+  "license": "Apache 2.0",
   "main": "exports/spaniel.js",
   "scripts": {
     "test": "npm run build && testem ci && node test/headless/run",
     "serve": "node test/headless/server/app",
-    "watch": "node watch",
+    "watch": "broccoli-timepiece exports",
```

Comment:
@krisselden I assume you meant to comment on the uuid bit? I've fixed that to be a sequence.